ee/modules/500-operator-trivy/openapi/config-values.yaml

```diff
@@ -84,3 +84,12 @@ properties:
     x-example:
       - app
       - env
+  insecureRegistries:
+    type: object
+    additionalProperties:
+      type: string
```

Comment:
I'm totally can't understand what is `key` here. Is it some abstract string?
I guess it would be better to make it an array of strings. And use array index as `key`

Reply:
Yes, it is just abstract: https://github.com/aquasecurity/trivy-operator/blob/18e40db60a957cf233afeb442bddca62a03e45e9/pkg/plugins/trivy/config.go#L461

Reply:
i replaced it:

<img width="277" alt="Screenshot 2024-10-16 at 12 54 39 AM" src="https://github.com/user-attachments/assets/92916985-43bc-4d8c-8fca-c79225df88a4">